Update acknowledgements (#37)
psychoinformatics-de/rdm-course: README.md

- [Project structure](https://slides.djnavarro.net/project-structure) by Danielle Navarro
- [The Turing Way](https://the-turing-way.netlify.app/) (CC-BY 4.0) 

Acknowledgements
================

> This DataLad extension was developed with funding from the Deutsche

Reviewer: ```suggestion
These training materials were developed with funding from the Deutsche
```
Maybe this way, as it's not really an extension?
